receiver/filereceiver/go.mod

```
@@ -0,0 +1,40 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filereceiver
+
+go 1.18
+
+require (
+	github.com/stretchr/testify v1.8.1
+	go.opentelemetry.io/collector v0.69.0
+	go.opentelemetry.io/collector/component v0.69.0
+	go.opentelemetry.io/collector/confmap v0.69.0
+	go.opentelemetry.io/collector/consumer v0.69.0
```

```suggestion
	go.opentelemetry.io/collector/component v0.70.0
	go.opentelemetry.io/collector/confmap v0.70.0
	go.opentelemetry.io/collector/consumer v0.70.0
```

You'll need to run `make tidy` after this.